src/features/Dashboard/components/WalletBreakdown/BreakdownChart.tsx

```diff
@@ -137,38 +140,41 @@ const BreakdownChart = React.memo(
     }));
     return (
       <MainWrapper>
-        <PieChart
-          width={400}
-          height={350}
-          onMouseLeave={() => handleMouseLeave(-1)}
-          onMouseEnter={() => handleMouseLeave(-1)}
-        >
-          <Pie
-            isAnimationActive={shouldAnimate}
-            activeIndex={selectedAssetIndex}
-            activeShape={ActiveSection}
-            data={displayBalances}
-            cx={200}
-            cy={200}
-            innerRadius={0}
-            outerRadius={110}
-            label={(p) => CustomLabel(p as CustomLabelProps, selectedAssetIndex, handleMouseOver)}
-            labelLine={false}
-            dataKey="fiatValue"
-            onMouseEnter={(_: any, index: number) => handleMouseOver(index)}
+        {isScanning && <SkeletonLoader type="wallet-chart" width={400} height={350} />}
+        {!isScanning && (
+          <PieChart
+            width={400}
+            height={350}
             onMouseLeave={() => handleMouseLeave(-1)}
-            animationDuration={800}
-            onAnimationStart={() => setIsChartAnimating(true)}
-            onAnimationEnd={() => {
-              setShouldAnimate(false);
-              setIsChartAnimating(false);
-            }}
+            onMouseEnter={() => handleMouseLeave(-1)}
```

Comment:
What does this ?

Reply:
No idea, I did not change it

Reply:
Oh the line is highlighted green, I assumed this is something you added x) Nevermind then :)